h/zos.h

```diff
@@ -1529,6 +1529,10 @@ DSAB *getDSAB(char *ddname);
 
 int dsabIsOMVS(DSAB *dsab);
 
+void message2(char *message);
```

Comment:
Without doing this, we get
` IEW2278I B352 INVOCATION PARAMETERS -
          TERM=YES,PRINT=NO,MSGLEVEL=4,STORENX=NEVER,LIST=NOIMP,XREF=YES,MAP=YE
          S,PRINT=YES,MSGLEVEL=0,ac=1,rent,case=mixed,map,xref,reus
 IEW2456E 9207 SYMBOL message2 UNRESOLVED.  MEMBER COULD NOT BE INCLUDED FROM
          THE DESIGNATED CALL LIBRARY.
 IEW2665S 40FF MODULE ZWESIS01  IS NON-EXECUTABLE AND WAS NOT SAVED BECAUSE
          STORENX=NEVER.
 IEW2008I 0F03 PROCESSING COMPLETED.  RETURN CODE =  12.
IEW5033 The binder ended with return code 12.`

Reply:
looks like because its used in metalio.c because metalio has a different set of functions that do wto for a different purpose than wtoPrintf was, in particular vfprintf.
I dont want to move that without knowing its purpose.

Reply:
I don't think `message` was supposed to be global in the first place, and it's not used anywhere outside `metalio.c` Perhaps it's safe to move it to `zos.c` entirely and not have `message2`?

Reply:
It's not safe @ifakhrutdinov because message( ) is also used in vfprintf( ) of metalio.c , a method that isn't being moved to zos.c 

so either message2 stays or I can move message entirely to zos.c but we make it not static , and declare it in the header, if I'm understanding C correctly

Reply:
For now, I'll keep message2 unless you think it's better otherwise

Reply:
* `message()` wasn't supposed to be global
* `message2()` just re-enforces that flaw by also being global
* you could move `message()` and keep it global, that `vsnprintf` would probably still be able use it, wouldn't it?

1. If you want to keep `message()` in metalio, at least call `message2()` something more descriptive, like `wtoMessage()`. `message2()` is just a bad name for a global library function.

2. Also, make the arguments in `message2()` (or `wtoMessage`) and `wtoPrintf2()` `const`, it's good practice because people will be able to pass constant literals without producing a compiler warning
```c
void message2(const char *message);
void wtoPrintf3(const char *formatString, ...);
```
Use `const` in the definitions too.

Reply:
Makes sense 👍